[Constellation] GS audit (#8899)
cloudflare/cloudflare-docs: content/constellation/get-started/first-constellation-worker.md

In your `image-classifier-worker`, find your [`wrangler.toml`](/workers/wrangler/configuration/) file.

Bindings allow your Workers to interact with resources on the Cloudflare developer platform, such as Constellation. Create a [binding](/constellation/platform/wrangler/#bindings) between your `image-classifier` Constellation project and your `image-classifier-worker` Worker in your `image-classifier-worker` Worker's `wrangler.toml` configuration file.

> Substitute the `project_id` with the one generated when you ran `npx wrangler constellation project list` in [Create a new Constellation project](/constellation/get-started/first-constellation-worker/#configure-your-project):

Author: ```suggestion
Substitute the `project_id` with the project ID generated after running `npx wrangler constellation project list` in [Create a new Constellation project](/constellation/get-started/first-constellation-worker/#configure-your-project):
```
